components/x-interaction/src/InteractionRender.jsx

```diff
@@ -1,21 +1,13 @@
 import { h } from '@financial-times/x-engine';
-import getComponentName from './concerns/get-component-name';
 import { loading } from './concerns/symbols';
-import { addSerialisationData } from './concerns/serialiser';
-import shortId from '@quarterto/short-id';
 
 export const InteractionRender = ({
+	id,
 	actions,
 	state,
 	initialState,
+	[loading]: isLoading,
```

Comment:
Does this need to be a symbol? Or to put it another way, do you expect consumers of this class to access this property?

Reply:
a) there's no way to access this externally, it's just so anyone using react devtools or whatever can see this is the internal one, b) #170 overhauls this logic anyway?